### DISCUSSION TITLE
How to add sampling to "RuntimeBatchRequest"

### DISCUSSION THREAD

**Discussion by MarekMath** [great-expectations/great_expectations#5600] [General]
Hello,

I have been exploring GE library last few days and found out that RuntimeBatchRequest is optimal solution for me. Can you please advise me on how to add parameter of sampling into the batch? 

In another function creating batches I have seen this parameter but id doesnt fit function which I use.

```
    "sampling": {
        "limit": 1000,
        "sample": 0.1
    }
```

Thank you for advise.

**Comment by talagluck** [comment]
Hi @MarekMath - it is not currently possible to use sampling together with a RuntimeBatchRequest. This is because if you are using a RuntimeBatchRequest, you will be using either a dataframe or a query, and so you should be able to take care of the sampling directly with your data prior to using Great Expectations for validation.

**Reply by MarekMath** [reply]
Thanks for your answer, I figured out later too that sampling before using GE may be option as well.
